docs/release-notes/release-dec-2019.md

```diff
@@ -0,0 +1,32 @@
+# December 2019 Release
+
+## Concurrency and Synchronization Support 
+
+Concurrency and Synchronization are supported in this release. Specifically, the following changes are made to JLang compiler:
+
+- `thread.start()` is implemented so that a program could run multiple threads at the same time.
+- `synchronized` code block and `synchronized` method signature are translated in the compiler frontend.
+- A monitor is implemented in the runtime to acquire and release a lock.
+- All runtime code is modified to be thread-safe.
+- Java condition variables such as `object.wait()` and `object.notify()` are now supported.
+
+An example multi-threading program is located at `tests/misc/Eratosthenes.java` to demonstrate JLang's parallelism. It implements [Sieve of Eratosthenes](https://en.wikipedia.org/wiki/Sieve_of_Eratosthenes), a paralell algorithm to find all prime numbers up to any given limit.
+
+### Threading
+
+Every Java thread is backed by one native thread after it starts. The main Java thread is running on the main native thread. Java daemon and non-daemon threads are respected by JLang runtime so that the program will only exit if all non-daemon threads terminates. Thread `yield` and `interrupt` are also implemented with a caveat that the current threading model is cooperative. In other words, `interrupt` could not force another thread to yield but the other thread will throw an `InterruptedException`  the next time `wait` or `join` is invoked.
```

Comment:
```suggestion
Every Java thread is backed by one native thread after it starts. The main Java thread is running on the main native thread. Java daemon and non-daemon threads are respected by JLang runtime so that the program will only exit if all non-daemon threads terminates. Thread `yield` and `interrupt` are not intended as first-class scheduling primitives. In other words, `interrupt` will not force another thread to yield but the other thread will throw an `InterruptedException`  the next time `wait` or `join` is invoked. This is consistent with the Java Thread API.
```